src/loader.c

```diff
@@ -127,6 +127,35 @@ static char * cstr_get_pair(char *p,
 	return cstr_skip_ws(p);
 }
 
+static bool strings_equal(const char *str_left, const char *str_right)
+{
+	if (!str_left != !str_right)
+		return false;
+	
+	return !str_left == !str_right || strcmp(str_left, str_right) == 0;
```

Comment:
I think this function should be optimized for readability a bit more instead of for succinctness.
```suggestion
	if (str_left == NULL && str_right == NULL)
		return true;
	
	if (str_left == NULL || str_right == NULL)
	    return false;
	
	return strcmp(str_left, str_right) == 0;
```

Reply:
done with commit [a111808](https://github.com/pgbouncer/pgbouncer/pull/764/commits/a111808a8910e8a14eb0eb57992d198cb0f75c26)